librocco_receipt_printer/constants.py

```diff
@@ -1,3 +1,7 @@
+import os
+
 COUCH_SERVER = "http://admin:admin@localhost:5000"
 
 VERSION = "v1"
+
+PRINT_OUTPUT_FILE = os.environ.get("PRINT_OUTPUT_FILE")
```

Comment:
I propose (changing my mind from what I wrote above) to default this to a temporary directory that is printed at program startup.

```
import tempfile
PRINT_OUTPUT_FILE = os.environ.get("PRINT_OUTPUT_FILE", tempfile.mkdtemp(prefix="prints", dir="/tmp"))
```